packages/core/lib/knexion-core.module.ts

```diff
@@ -23,7 +23,7 @@ import { KnexionTransactionService } from './services';
 @Module({})
 export class KnexionCoreModule implements OnApplicationShutdown {
   static forRoot(options: KnexionModuleOptions = {}): DynamicModule {
-    const typeOrmModuleOptions = {
+    const knexModuleOptions = {
```

Comment:
Our module has name `knexion`, but not `knex`. Let's also find other places where we accidentally use `knex`